packages/terra-navigation-prompt/docs/README.md

```diff
@@ -100,10 +100,14 @@ The keys expected in the resolvePrompts Object or return value include:
 |Key|Type|Description|
 |---|---|---|
 |`title`|String|The title of the NotificationDialog.|
-|`message`|String|The message of the NotificationDialog.|
-|`rejectButtonText`|String|The text to render within the NotificationDialog's primary action button.|
-|`acceptButtonText`|String|The text to render within the NotificationDialog's secondary action button.|
-|`emphasizedAction`|String|A string indicating which action should be rendered as the NotificationDialog's primary action. Supported values include: `accept`, `reject`. Defaults to `accept`.|
+|`message`|String|Deprecated, The message of the NotificationDialog. Resolves to 'startMessage'|
```

Comment:
Would we be able to switch this to MDX and just import the propsTable from notification dialog?  It would be really nice if that would work so we wouldn't have to manually update this file.

Reply:
We probably could but I'm not sure if we want to. Currently this prop table doesn't expose the   `onAccept` or `onReject` functions that get applied to notification dialog, because I believe navigation prompt resolves those itself. 

Tagging @tbiethman to double check me here.

Reply:
Boo  👎 Okay, that's fair.

Reply:
@BenBoersma Yeah the NavigationPromptCheckpoint would convert onAccept/onReject into a resolve/reject on the Promise it returns to consumers. So it's close, but not quite accurate to use the other's propsTable directly.